build.gradle

```diff
@@ -15,6 +16,7 @@ buildscript {
         classpath 'com.android.tools.build:gradle:7.0.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
         classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.6.10"
+        classpath "de.mannodermaus.gradle.plugins:android-junit5:1.8.2.0"
```

Comment:
This plugin configures test tasks to execute junit5 correctly. We could try not to add this dependency, but considering it was only going to be used for tests and it was popular enough, I thought it was ok.